### DISCUSSION TITLE
Crop suffix of some dimension using `SpatialCrop`

### DISCUSSION THREAD

**Discussion by function2-llx** [Project-MONAI/MONAI#1935] [Q&A]
Hello everyone. For some purpose I'd like to use the last 16 slices of my each MRI scan. For a numpy array `x` this can be done with `x[0:None, 0:None, -16:None]`. However currently I'm not able to find a way to achieve this with MONAI's `SpatialCrop`. In this case I need to pass `roi_start=(0, 0, -16)` and `roi_end=(None, None, None)` to construct `SpatialCrop`, but this leads to an exception raised [here](https://github.com/Project-MONAI/MONAI/blob/master/monai/transforms/croppad/array.py#L244) saying that:
```
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```
While simple workaround by slicing it outside, or even using the transform of `Lambda`, is obvious, but it will be pretty cool if I could process my images only with transforms provided by MONAI, right? 

Any suggestion will be appreciated!

**Comment by function2-llx** [comment]
Seems duplicate of #1929, I misunderstood that question at first. 

Sorry about that, please refer to that discussion.

**Comment by rijobro** [comment]
Hi, suggested PR [here](https://github.com/Project-MONAI/MONAI/pull/1954). If you wanted to leave the last 16 in the z-direction and not touch anything else, this would be done with:

```python
SpatialCrop(roi_slices=[slice(None), slice(None), slice(-16, None)])
```
